crates/tools/src/main.rs

```diff
@@ -348,6 +349,55 @@ async fn run_cli() -> Result<()> {
                         .help("l1 sudt script hash, default for withdrawal CKB"),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("withdraw-to-v1")
+                .about("withdraw CKB / sUDT from godwoken v0 to v1")
+                .arg(arg_privkey_path.clone())
+                .arg(arg_deployment_results_path.clone())
+                .arg(arg_config_path.clone())
+                .arg(arg_godwoken_rpc_url.clone())
+                .arg(
+                    Arg::with_name("capacity")
+                        .short("c")
+                        .long("capacity")
+                        .takes_value(true)
+                        .required(true)
+                        .help("CKB capacity to withdrawal"),
+                )
+                .arg(
+                    Arg::with_name("amount")
+                        .short("m")
+                        .long("amount")
+                        .takes_value(true)
+                        .default_value("0")
+                        .help("sUDT amount to withdrawal"),
+                )
+                .arg(
+                    Arg::with_name("fee")
+                        .short("f")
+                        .long("fee")
+                        .takes_value(true)
+                        .default_value("0.0001")
+                        .help("Withdrawal fee, default to 0.0001 CKB"),
+                )
+                .arg(
+                    Arg::with_name("eth-address")
+                        .long("eth-address")
+                        .takes_value(true)
+                        .required(true)
+                        .help("v1 eth address"),
+                )
+                .arg(
+                    Arg::with_name("sudt-script-hash")
+                        .long("sudt-script-hash")
+                        .takes_value(true)
+                        .required(false)
+                        .default_value(
+                            "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
Is this default value valid? I suggest we remove the default value if it is invalid.

Reply:
Its same as "withdraw" command. 

https://github.com/nervosnetwork/godwoken/blob/43b7427b3a40d85ee439285865863fec5a057648/crates/tools/src/main.rs#L347